pkgs/development/python-modules/dunamai/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, poetry-core
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, importlib-metadata
+, packaging
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "dunamai";
+  version = "1.10.0";
+  format = "pyproject";
```

Only Python > 3.5 is [supported](https://github.com/mtkennerly/dunamai/blob/master/pyproject.toml#L20). As all 
 Python releases < 3.7 are EOL, I suggest to set it 3.7 (can always be adjusted if one confirm that the module works.). 

```suggestion
  format = "pyproject";

  disabled = pythonOlder "3.7";
```

---

Is it pretty standard to enforce minimum versions by disabling the package? Just trying to learn proper decorum for my future contributions :) Thanks for the tip.